tasks/main.yml

```diff
@@ -1,3 +1,7 @@
+- assert:
+    that: "'{{ ansible_host | ipaddr }}' == '{{ ansible_host }}'"
```

Comment:
This should actually check tha `LOAD_BALANCER_SERVER_IP` is an IP address, since this is what we actually care about.  `LOAD_BALANCER_SERVER_IP` might be overridden with an IP, and then it doesn't matter whether `ansible_host` is an IP or not.  (Sorry that I missed that last time.)